apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/ReadTests.cs

```diff
@@ -288,5 +331,16 @@ public class ReadTests : IClassFixture<TestDatabaseFixture> {
         //    Assert.NotNull(exceptionCaught);
         //    Assert.Equal(0, result);
         //}
+
+        private async Task<T> ExecuteAsync<T>(string sql)
+        {
+            T result;
+            using (var connection = await _testFixture.GetTestDatabaseConnectionAsync())
+            {
+                var cmd = connection.CreateSelectCommand(sql);
+                result = await cmd.ExecuteScalarAsync<T>();
```

Comment:
usually that rule is for increasing readability of large methods and this guy is about 5 lines.  But sure, I've rearranged the lines.